src/UserSettingsStore.js

```diff
@@ -112,4 +112,14 @@ module.exports = {
             append: true,  // We always append for email pushers since we don't want to stop other accounts notifying to the same email address
         });
     },
+
+    isFeatureEnabled: function(feature: string): boolean {
+        feature = feature.match(/\w+/g).join('_').toLowerCase();
```

Comment:
Not sure I'm a big fan of the magic here - I'd prefer `LABS_FEATURES` to just be an object.